tests/portfolio/test_ledger_entries.py

```diff
@@ -0,0 +1,43 @@
+import os
+import pytest
+from playwright.sync_api import Page, expect
+from vega_sim.service import VegaService
+from datetime import datetime, timedelta
+from conftest import init_vega
+from fixtures.market import setup_continuous_market
+
+@pytest.fixture(scope="module")
+def vega(request):
+    with init_vega(request) as vega:
+        yield vega
+
+
+@pytest.fixture(scope="module")
+def continuous_market(vega):
+    return setup_continuous_market(vega)
+
+
+@pytest.mark.usefixtures("page", "auth", "risk_accepted")
+def test_ledger_entries_downloads(continuous_market, vega: VegaService, page: Page):
```

Comment:
```suggestion
def test_ledger_entries_downloads(continuous_market,  page: Page):
```

It doesn't look like we are using vega in the test.